src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -1012,25 +1017,24 @@ namespace AirflowNetwork {
 #endif
     }
 
-    int GenericCrack(EnergyPlusData &state,
-                     Real64 &coef,               // Flow coefficient
-                     Real64 const expn,          // Flow exponent
-                     bool const LFLAG,           // Initialization flag.If = 1, use laminar relationship
-                     Real64 const PDROP,         // Total pressure drop across a component (P1 - P2) [Pa]
-                     const AirProperties &propN, // Node 1 properties
-                     const AirProperties &propM, // Node 2 properties
-                     std::array<Real64, 2> &F,   // Airflow through the component [kg/s]
-                     std::array<Real64, 2> &DF   // Partial derivative:  DF/DP
+    void generic_crack(Real64 &coefficient,        // Flow coefficient
+                       Real64 const exponent,      // Flow exponent
+                       bool const linear,          // Initialization flag. If true, use linear relationship
+                       Real64 const pdrop,         // Total pressure drop across a component (P1 - P2) [Pa]
+                       const AirProperties &propN, // Node 1 properties
+                       const AirProperties &propM, // Node 2 properties
+                       std::array<Real64, 2> &F,   // Airflow through the component [kg/s]
+                       std::array<Real64, 2> &DF   // Partial derivative:  DF/DP
```

Comment:
Modify the generic crack function so that it computes in the same way as the SurfaceCrack object.